DuckDuckGo/Updates/ReleaseNotesUserScript.swift

```diff
@@ -77,7 +77,10 @@ final class ReleaseNotesUserScript: NSObject, Subfeature {
             return
         }
 
-        let updateController = Application.appDelegate.updateController
+        guard let updateController = Application.appDelegate.updateController else {
+            return
+        }
```

Comment:
Just to satisfy the compiler as AppDelegate.updateController is defined as `UpdateController!`